module/programs/helper.js

```diff
@@ -1075,6 +1136,8 @@ module.exports = class ProgramsHelper {
 						externalId: program.externalId,
 						_id: program._id,
 						isAPrivateProgram: program.isAPrivateProgram,
+						ongoingProjects: program.ongoingProjects ? program.ongoingProjects : 0,
+						completedCount: program.completedCount ? program.completedCount : 0,
```

Comment:
@praveenKDass please make this completedProjects

Reply:
Changed @VISHNUDAS-tunerlabs